test/typescript/apis/project/cds-services.ts

```diff
@@ -82,9 +82,9 @@ await srv.run([query, query])
 await srv.run('SELECT * from Authors where name like ?', ['%Poe%'])
 await srv.run('SELECT * from Authors where name like :name', { name: '%Poe%' })
 
-srv.foreach({ SELECT: { from: { ref: ['Foo'] } } }, () => {})
+srv.foreach({ SELECT: { from: { ref: ['Foo'] } }, elements: {} }, () => {})
```

Comment:
I am not sure if this test case is relevant, why the elements are defined here ? It is also not documented https://cap.cloud.sap/docs/node.js/events#query

Reply:
You are right, this is missing from CAPire. It is mentioned in [the relevant changelog](https://pages.github.tools.sap/cap/docs/releases/archive/2023/nov23#inferring-query-elements). Could you please forward this gap internally to have the maintainer responsible for this change update the docs where you found it to be missing? Thanks!